app.js

```diff
@@ -1,93 +1,109 @@
 'use strict';
-var middleName = prompt('Can you guess if my middle name is Charles? \(Yes or No\)');
+
+function middleName() {
+  var middleName = prompt('Can you guess if my middle name is Charles? \(Yes or No\)');
   middleName.toLowerCase();
```

Comment:
this is not actually mutating the value of middleName.  .toLowerCase() returns the lowercased value, but doesn't actually modify the original string